### ISSUE TITLE
Nornir 3.0: log_file name specified in config.yaml is not honored

### ISSUE THREAD

**Issue by ktbyers** [nornir-automation/nornir#561]
nornir.log is always used even when an alternate file_name is specified:

```
$ cat config.yaml 
inventory:
  plugin: SimpleInventory
  options:
    host_file: "~/nornir_inventory/hosts.yaml"
    group_file: "~/nornir_inventory/groups.yaml"
    defaults_file: "defaults.yaml"

logging:
  enabled: True
  level: DEBUG
  file: my_log.log
```

```
# directory state before executing script
$ ls -ltr
total 16
-rw-rw-r-- 1 kbyers kbyers  22 Apr 17 10:59 defaults.yaml
-rw-rw-r-- 1 kbyers kbyers 239 Jun 12 11:12 config.yaml
-rw-rw-r-- 1 kbyers kbyers 934 Jun 16 14:24 exercise3b.py
-rw-rw-r-- 1 kbyers kbyers 661 Jun 16 14:24 exercise3a.py
```

```
# nornir.log is created instead of the expected 'my_log.log'
$ python exercise3a.py 
$ ls -ltr
total 20
-rw-rw-r-- 1 kbyers kbyers  22 Apr 17 10:59 defaults.yaml
-rw-rw-r-- 1 kbyers kbyers 239 Jun 12 11:12 config.yaml
-rw-rw-r-- 1 kbyers kbyers 934 Jun 16 14:24 exercise3b.py
-rw-rw-r-- 1 kbyers kbyers 661 Jun 16 14:24 exercise3a.py
-rw-rw-r-- 1 kbyers kbyers 169 Jun 16 14:54 nornir.log
```

This works properly on Nornir 2.X

**Comment by ktbyers** [comment]
Still was failing with using new name of `log_file`

```
logging:
  enabled: True
  level: INFO
  log_file: output.txt
  to_console: True
```

Environment variable worked fine:
export NORNIR_LOGGING_LOG_FILE=foo.txt

**Comment by ktbyers** [comment]
Relevant Bug-fix is here:

https://github.com/nornir-automation/nornir/pull/564

**Comment by dbarrosop** [comment]
Closed bia #564
